packages/calling/src/CallingClient/types.ts

```diff
@@ -27,10 +28,82 @@ export type CallingClientErrorEmitterCallback = (
   finalError?: boolean
 ) => void;
 
+/**
+ * Interface for `CallingClient` module.
+ * `CallingClient` module is designed to offer set of APIs that are related to doing line registration, calling functionalities on the SDK
+ */
 export interface ICallingClient extends Eventing<CallingClientEventTypes> {
+  /**
+   * Represents the `mediaEngine for managing media-related operations within the CallingClient.
+   * The media engine provides access to audio and video devices such as cameras, microphones, and speakers within the media layer.
+   *
+   * @public
+   * @example
+   * ```
+   * const microphones = await callingClient.mediaEngine.Media.getMicrophones();
+   * const speakers = await callingClient.mediaEngine.Media.getSpeakers();
+   * const cameras = await callingClient.mediaEngine.Media.getCameras();
+   * ```
+   */
   mediaEngine: typeof Media;
-  getSDKConnector: () => ISDKConnector;
-  getLoggingLevel: () => LOGGER;
-  getActiveCalls: () => Record<string, ICall[]>;
-  getConnectedCall: () => ICall | undefined;
+  getSDKConnector(): ISDKConnector;
+  getLoggingLevel(): LOGGER;
+
+  /**
+   * Retrieves details of the line object(s) belonging to a user.
+   *
+   * @returns A dictionary where each key is a line identifier and the corresponding
+   *          value is the {@link ILine} object associated with that identifier.
+   *
+   * @example
+   * ```typescript
+   * const lines = callingClient.getLines();
+   * // {
+   * //   'lineId1': lineObj1,
+   * //   'lineId2': lineObj2,
+   * // }
+   */
+  getLines(): Record<string, ILine>;
+
+  /**
+   * Retrieves a dictionary of active calls grouped by `lineId`.
+   *
+   * This method gathers active call objects and organizes them into a dictionary
+   * where keys represent `lineId`s and values are arrays of active calls associated
+   * with each line.
+   *
+   * @returns A dictionary where each key is a `lineId` and the corresponding value
+   *          is an array of {@link ICall} objects that are active and associated with that line.
+   *
+   * @example
+   * ```typescript
+   * const activeCalls = callingClient.getActiveCalls();
+   * // {
+   * //   'line1': [call1, call2],
+   * //   'line2': [call3],
+   * // }
+   * ```
+   */
+  getActiveCalls(): Record<string, ICall[]>;
+
+  /**
+   * Retrieves the call object for the currently connected call in the client.
+   *
+   * This method iterates through active call objects and returns the call
+   * that is currently connected (not on hold).
+   *
+   * @returns The {@link ICall} object for the connected call, or undefined if no connected
+   *          call is found.
+   *
+   * @example
+   * ```typescript
+   * const connectedCall = callingClient.getConnectedCall();
+   * if (connectedCall) {
+   *   console.log(`Connected call ID: ${connectedCall.callId}`);
```

Comment:
Remove console logs, just say call object for the connected call will be returned. If you need to give some reference then add an example of ICall whose status will reflect connected